src/ansys/simai/core/data/post_processings.py

```diff
@@ -274,6 +275,40 @@ class SurfaceVTP(_PostProcessingVTKExport):
     """
 
 
+class PostProcessingOnCells(SurfaceVTP):
```

Comment:
I didn't follow the product story but these class name seems too generic. Maybe `SurfaceOnCells` and `SurfaceAsLearnt` ?